### PR TITLE
Replace ko-fi banner with quick filter

### PR DESCRIPTION
**Pull request by Mitcheljager**
This replaces the donation banner with a quick filter to quickly go to some of the more popular sort pages. These pages could be accessed before through the filter, but this adds an easier to find alternative.

image

## Desktop
image

## Tablet-ish
image

## Mobile
image

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -10,13 +10,8 @@
 
     <%= render "on_fire_carousel" %>
 
-    <%= link_to on_fire_path, class: "button button--secondary" do %>
-      <%= inline_svg_tag "icons/icon-onfire.svg", class: "mr-1/8", height: 16, width: 16 %>
-      <%= t(".on_fire.button") %>
-    <% end %>
-
     <div class="logo-background logo-background--overwatch">
-      <%= render "application/banners/ko_fi" %>
```
Comment (CactusPuppy) [inline app/views/posts/index.html.erb:19]: Rest in peace Ko-Fi banner ;-;

File: app/views/filter/_quick_filter.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <% end %>
+
+  <%= link_to filter_path(sort: :views), class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-views.svg", class: "quick-filter__icon" %>
+    Most Views
+  <% end %>
+
+  <%= link_to filter_path(sort: :updated), class: "quick-filter__item" do %>
+    <%= inline_svg_tag "icons/icon-clock.svg", class: "quick-filter__icon" %>
+    Last Updated
```
Comment (CactusPuppy) [inline app/views/filter/_quick_filter.html.erb:19]: Consider adding an accessible description to these SVGs with a `title:` or `desc:`. More information can be found at https://github.com/jamesmartin/inline_svg#options.

**Review by CactusPuppy — CHANGES_REQUESTED** [review]
Looks good overall! Just be sure to add some accessible descriptions or titles to the SVG :)

File: app/views/filter/_quick_filter.html.erb (hunk 2 above)
Comment (Mitcheljager) [reply app/views/filter/_quick_filter.html.erb]: Good call!

**Pull request merged**